### PR TITLE
support simple webvtt color tags

### PR DESCRIPTION
**Pull request by adipose**
supports foreground and background color.  depending on mpc-hc settings, bg may be an outline rather than a flat rectangle, but it's still better than losing the bg entirely.

### CODE REVIEW

**Comment by clsid2** [comment]
There seems to be something wrong with the commit. Github shows diff for last commit on develop branch somehow.

**Comment by adipose** [comment]
fixed.

File: src/Subtitles/STS.cpp [hunk 1]
```diff
@@ -487,6 +500,8 @@ static CStringW WebVTT2SSA(CStringW str)
         str.Replace(L"</b>", L"{\\b}");
         str.Replace(L"<u>", L"{\\u1}");
         str.Replace(L"</u>", L"{\\u}");
+        str.Replace(L"<font color=", L"{c}");
+        str.Replace(L"</i>", L"{\\i}");
```
Comment (ale5000-git) [inline src/Subtitles/STS.cpp:504]: @adipose: Shouldn't there be `</font>` instead of `</i>`?

Comment (adipose) [reply src/Subtitles/STS.cpp:504]: Actually these lines should be removed. I forgot to clean them up.

**Pull request merged**